opennode.php

```diff
@@ -189,19 +189,19 @@ public function payment_callback()
                         }
                         break;
                     case 'processing':
-                        $order->add_order_note(__('Customer has paid via standard on-Chain. Payment is awaiting 1 confirmation by the Bitcoin network, DO NOT SEND purchased goods/services UNTIL payment has been marked as PAID', 'opennode'));
+                        $order->add_order_note(__('Customer has paid via standard on-Chain. Payment is awaiting 1 confirmation by the Bitcoin network, DO NOT SEND purchased goods/services UNTIL payment has been marked as PAID', 'opennode-for-woocommerce'));
                         break;
                     case 'underpaid':
                         $missing_amt = number_format($cgOrder->missing_amt/100000000, 8, '.', '');
-                        $order->add_order_note(__('Customer has paid via standard on-Chain, but has underpaid by ' . $missing_amt . ' BTC. Waiting on user to send the remainder before marking as PAID.', 'opennode'));
+                        $order->add_order_note(__('Customer has paid via standard on-Chain, but has underpaid by ','opennode-for-woocommerce') . $missing_amt . __(' BTC. Waiting on user to send the remainder before marking as PAID.', 'opennode-for-woocommerce'));
                         break;
                     case 'expired':
-                      $order->add_order_note(__('Payment expired', 'opennode'));
+                      $order->add_order_note(__('Payment expired', 'opennode-for-woocommerce'));
                       $order->update_status('cancelled');
                       break;
                     case 'refunded':
                         $refund_id = $cgOrder->refund['id'];
-                        $order->add_order_note(__('Customer has canceled the payment. Refund ID - ' . $refund_id . ' .', 'opennode'));
```

Comment:
with text-domain being anything other than `opennode-for-woocommerce`, translate.wp.org does not pick up string as translatable